test/k8s/Dockerfile

```diff
@@ -17,5 +17,5 @@ ADD dist/bin/* /usr/local/bin/
 ARG RUNTIME
 RUN cat <<EOF >> /etc/containerd/config.toml
 [plugins."io.containerd.grpc.v1.cri".containerd.runtimes.wasm]
-runtime_type = "io.containerd.${RUNTIME}.v1"
+runtime_path = "io.containerd.${RUNTIME}.v1"
```

Comment:
I didn't fully follow the discussion in Slack about this, but I would expect runtime_path to be a path, while this is not a path.

Reply:
I don't understand why the kind test fails.
This is what I wanted to do. Set runtime_type to path.
Is this difference okay?

https://github.com/containerd/runwasi/blob/472f431542f5a7e37168bc7dd974d011f56ad207/test/k3s/bootstrap.sh#L8
https://github.com/containerd/runwasi/blob/472f431542f5a7e37168bc7dd974d011f56ad207/test/k8s/Dockerfile#L20

Reply:
Why is runtime_type a path in k3s but not in k8s? I am confused. 

Should `runtime_type` always be in the form of `io.containerd.${RUNTIME}.{version}` for k8s and k3s?

Reply:
Found it. It's probably mentioned here.
https://github.com/containerd/containerd/blob/v2.0.2/core/runtime/runtime.go#L49-L51
And it is processed by resolveRuntimePath.
https://github.com/containerd/containerd/blob/v2.0.2/core/runtime/v2/shim_manager.go#L308-L384